careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
 - Works in other areas of the codebase with guidance from owners of that system.
+- Owns a service or large technology component.
+
+#### Impact
+- Delivers significant business value at team level.
+- Helps the team deliver on its overall goals.
+- Can drive specific initiatives.
+
+#### Influence
 - Persistent in the face of roadblocks, dispatches them efficiently, pulling in others as necessary.
-- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across
-  projects.
+- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across projects.
+- Effectively partners with team's cross-functional partners (including PM, Designer, Data Analyst).
 - Mentors junior engineers.
+
+#### Discretion
+- Requires minimal direction or oversight, delivers quickly.
+- Understands elements of the business that their work supports.
+- Demonstrates fluency with team KPIs and key drivers of these KPIs.
 - Demonstrates knowledge of industry trends, infrastructure and systems.
 
 ### IC4 Staff Engineer
 
+#### tl;dr: can drive change and impact across multiple teams
+
+#### Knowledge Leadership
 - Owns large scale and impact service or collection of services.
-- Understands the Artsy business and makes decisions based on its direction.
-- Sought out by other team members for technical guidance.
+
+#### Impact
 - Anticipates technical issues at the product level and makes architectural and design decisions to avoid them.
-- Recognized as a prolific contributor.
-- Multiplies the effectiveness of others by facilitating cross-team work.
 - Has track record of creating sweeping improvements in stability, performance, and scalability across services.
-- Consistently reduces the complexity of projects, services, and processes in order to get more done with less
-  work.
+- Consistently reduces the complexity of projects, services, and processes in order to get more done with less work.
 - Has made an obvious positive impact on the entire company's technical trajectory.
+
+#### Influence
+- Sought out by other team members for technical guidance.
+- Recognized as a prolific contributor.
+- Multiplies the effectiveness of others by facilitating cross-team work.
+
+#### Discretion
+- Understands the Artsy business and makes decisions based on its direction.
 - Sets short to medium-term strategic technical direction.
 
 ### IC5 Senior Staff Engineer
 
-- Anticipates broad technical change and ensures Artsy is always ahead of the curve.
-- Consistently delivers large systems involving one or more teams' contribution.
-- Primarily acts as a multiplier by building systems, authoring tools, or introducing policies or patterns.
-- Quickly breaks down complex problems into potential solutions, knowns, and unknowns.
+#### tl;dr: can drive change and impact across full Product, Data, Design and Engineering organization
```

Comment:
This tl;dr sounds like a rewording of the IC4 tl;dr because "multiple teams" is the "full PDDE organization". Perhaps engineers at the IC5 level should be driving impact across the company?